doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1069,7 +1069,7 @@ one can define the addition function as :
    | S p => S (add p m)
    end.
 
-The ``{struct`` :token:`ident```}`` annotation may be left implicit, in this case the
+The ``{struct`` :token:`ident` ``}`` annotation may be left implicit, in this case the
```

Comment:
Note that this is already fixed in #7537 and the best fix goes further than this.